includes/Embed_Base.php

```diff
@@ -83,6 +83,15 @@ public function init() {
 		add_filter( 'wp_kses_allowed_html', [ $this, 'filter_kses_allowed_html' ], 10, 2 );
 	}
 
+	/**
+	 * Get the action priority to use for registering the service.
+	 *
```

Comment:
New methods like this would need `@since` annotations

Reply:
Fixed